addOns/help/src/main/javahelp/contents/start/proxies.html

```diff
@@ -113,6 +113,18 @@ <H3>OS X System Proxy settings</H3>
 <tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Press the</td><td> Network Options 'Apply' button</td></tr>
 </table>
 
+<H3>Linux Gnome Proxy settings</H3>
+
+<table>
+<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Select the</td><td> 'Gnome Settings'</td></tr>
+<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Select the</td><td> 'Network:'</td></tr>
```

Comment:
Control panel? Icon? Applet?

Reply:
Left side menu has been added to describe where to click.

I prefer to not add icon since this icon can change with new versions and make this guide more confusing. What do you think?